cmd/meroxa/global/config.go

```diff
@@ -23,16 +22,22 @@ func readConfig() (*viper.Viper, error) {
 		cfg.SetConfigFile(flagConfig)
 	} else {
 		// Find home directory.
-		home, err := homedir.Dir()
+		configDir, err := os.UserConfigDir()
+		if err != nil {
+			return nil, fmt.Errorf("could not get config directory: %w", err)
+		}
+		configDir = filepath.Join(configDir, "meroxa")
+
+		// create subdirectory if it doesn't exist, otherwise viper will complain
+		err = os.MkdirAll(configDir, 0755)
 		if err != nil {
-			return nil, fmt.Errorf("could not get home directory: %w", err)
+			return nil, fmt.Errorf("could not create meroxa config directory: %w", err)
 		}
 
-		// Set the base name of the config file, without the file extension.
-		cfg.SetConfigName(defaultConfigFilename)
-		cfg.AddConfigPath(home)
+		cfg.AddConfigPath(configDir)
+		cfg.SetConfigName("config")
```

Comment:
Maybe we should document this in the README.md, that meroxa configuration will be stored elsewhere now.